Godeps/Godeps.json

```diff
@@ -86,6 +86,10 @@
 			"Rev": "deed6eddef4f5e28c14d26d3c25b1fc454ce8baf"
 		},
 		{
+			"ImportPath": "github.com/davecgh/go-spew/spew",
+			"Rev": "2df174808ee097f90d259e432cc04442cf60be21"
```

Comment:
Updated to go-spew v1.1.1 and re-ran tests (they still pass).

Note: This dependency is only required for go-grpc-prometheus' tests. If we could exclude those somehow we could reduce the amount of stuff we need to pull in. At some point I thought godep was ignoring test files - any idea if that changed, or what?